airbyte-integrations/connectors/source-google-analytics-data-api/Makefile

```diff
@@ -0,0 +1,19 @@
+docker_image := airbyte/$(notdir $(CURDIR)):dev
+
+run-build:
+	docker build . -t ${docker_image}
+
+spec:
+	@docker run --rm $(docker_image) spec | jq
+
+check:
+	@docker run --rm -v $(PWD)/secrets:/secrets $(docker_image) check --config /secrets/config.json | jq
+
+discover:
+	@docker run --rm -v $(PWD)/secrets:/secrets $(docker_image) discover --config /secrets/config.json | jq
+
+read:
+	@docker run --rm -v $(PWD)/secrets:/secrets -v $(PWD)/integration_tests:/integration_tests $(docker_image) read --config /secrets/config.json --catalog /integration_tests/configured_catalog.json | jq
+
+unittest-local:
+	@python -m pytest unit_tests
```

Comment:
what's this Makefile for?

Reply:
It is not necessary to have it in repository. I created it for running routine operations more quickly. It is only for development purpose and for convenience. For example `check` is a long command but I am just typing `make check` in my console